modules/local/append_screen_hits/main.nf

```diff
@@ -0,0 +1,54 @@
+process APPEND_SCREEN_HITS {
+    tag "$meta.id"
+    label 'process_low'
+
+    conda "conda-forge::biopython=1.78 conda-forge::pandas=1.3.5"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/mulled-v2-80c23cbcd32e2891421c54d1899665046feb07ef:77a31e289d22068839533bf21f8c4248ad274b60-0' :
+        'quay.io/biocontainers/mulled-v2-80c23cbcd32e2891421c54d1899665046feb07ef:77a31e289d22068839533bf21f8c4248ad274b60-0' }"
```

Comment:
```suggestion
        'biocontainers/mulled-v2-80c23cbcd32e2891421c54d1899665046feb07ef:77a31e289d22068839533bf21f8c4248ad274b60-0' }"
```

the registries are provided separately in `nextflow.config`. could you plz confirm this?

Reply:
I just removed the prefix and the tests still passed!